src/show.c

```diff
@@ -14,7 +14,7 @@ void show_menu(char *message, char *shadow) {
         printw("I:%u %s", lines[cy].ident, filename);
         char buf[256];
         int len = snprintf(buf, 256, "%s %s",
-                           config.syntax_on && config.current_syntax ? config.current_syntax->name : "Off",
+                           config.current_syntax ? config.current_syntax->name : "Off", // check if current_syntax is NULL
```

Comment:
For now is better to don't remove this check because  if there's a bug that doesn't set correctly config.current_syntax, the problem will be easier to debug without a crash

Reply:
I think it would be better to add a check if `current_syntax` is NULL, and in the case of it being NULL, `fprintf(stderr, "FATAL: current_syntax is NULL"`. And `exit(1)`.

Reply:
This should be the perfect job for `assert`, but if we call exit without cleaning up raw mode and ncurses the terminal will be messed up (actually happens every time I found a bug)